src/components/contactForm/ContactForm.tsx

```diff
@@ -57,17 +62,28 @@ const ContactForm = () => {
 
     if (!values.name) {
       errors.name = "Required";
+    } else if (values.name.length < minInputLengths.minNameLength) {
```

Comment:
Probably better would be to replace this IF validation witch validation Schema which is used in this file. 
You can use those validators: 
  .min(2, 'Too Short!')
  .max(50, 'Too Long!')

Reply:
The validation schema is the same as in the Formik documentation. I think it is more readable than the one transferred to yup.

Reply:
I thought that you are using both Yup and conditional validation in one file. I see that there is only conditional one - in this case its ok for me to have it in this way but still validation way should be unified in all files.